docs/pages/faq/namespaceoffloading/index.md

```diff
@@ -0,0 +1,155 @@
+
+There are two types of *conditions*:
+
+**Ready condition**
+
+   | Value     | Description |
+   | -------   | ----------- |
+   | **True**  |  The namespace is successfully created inside the remote cluster. |
+   | **False** |  There was a problem during the remote namespace creation. |
+
+**OffloadingRequired condition**
+
+   | Value   | Description |
+   | ------- | ----------- |
+   | **True**  |  The remote namespace is requested to be created inside this cluster (the condition `OffloadingRequired = true` is removed when the remote namespace acquires a `Ready` condition). |
+   | **False** |  The remote namespace creation inside this cluster is not required. |
+
+{{% notice note %}}
+The *remoteNamespaceCondition* syntax is the same of the standard [NamespaceCondition](https://pkg.go.dev/k8s.io/api/core/v1@v0.21.0#NamespaceCondition).
+{{% /notice %}}
+
+#### OffloadingPhase field
+
+Another important field of the *namespaceOffloading* status is the *offloadingPhase*.
+It informs you about the global offloading status.
+
+The possible error values are:
+
+   | Value   | Description |
+   | ------- | ----------- |
+   | **AllFailed**  |  There was a problem during the creation of all remote namespaces. |
+   | **SomeFailed** |  There was a problem during the creation of some remote namespaces (one or more). |
+
+To have a more detailed description of the other values that the *offloadingPhase* can assume, look at the [NamespaceOffloading Status section](#).
+
+If instead you have selected some clusters, but the *offloadingPhase* is "**NoClusterSelected** ” then there may be an error in the specified *clusterSelector* syntax. 
+In this case, the *namespaceOffloading* resource should expose an annotation with key `liqo.io/scheduling-enabled` that signals the syntax error:
+
+```bash
+kubectl get namespaceoffloadings -n $NAMESPACE -o yaml | grep "liqo.io/scheduling-enabled:" -A 1
+```
+
+The *offloadingPhase* takes a few seconds to be updated, so initially, the value may not match the expected one.
+
+If, after a couple of seconds, the resource status is not correctly updated, then you can continue your troubleshooting.
+
+### NamespaceMap resource
+
+The *remoteNamespacesCondition* field seen above is a map that has "*remote cluster-id* " as its keys. 
+You have to:
+
+1. Identify the remote cluster that has problems.
+2. Get its cluster-id from the *remoteNamespacesCondition*.
+3. Enter the cluster-id in the following variable.
+      ```bash
+      REMOTE_CLUSTER_ID=<your-remote-cluster-id>
+      echo "Remote cluster-id: $REMOTE_CLUSTER_ID"
+      ```
+
+For each remote cluster, there is a "*Liqo Tenant namespace*" containing several Liqo CRDs.
+We are interested in the resource called *NamespaceMap*.
+This resource keeps track of:
+
+* All remote namespace creation requests (in the map “*Spec.DesiredMapping* ”).
+* The actual status of the remote namespaces (in the map “*Status.CurrentMapping* ”).
```

Comment:
Would personally move this before the steps above, as it gives the context of what you are actually doing.